alex/applications/PublicTransportInfoCS/hdc_policy.py

```diff
@@ -376,33 +376,36 @@ def get_platform_res_da(self, ds, ludait, slots_being_requested,
-                                                      platform_res.platform))
-                            res_da.append(DialogueActItem('inform', 'track',
-                                                      platform_res.track))
-                            if platform_info.train_name != 'none':
-                                res_da.append(DialogueActItem('inform',
-                                                              'train_name',
-                                                              platform_info.train_name))
+                                                          'not_found'))
+                            res_da.append(DialogueActItem('inform', 'direction',
+                                                          platform_info.to_stop))
+                        else:
+                            if platform_res.platform:
+                                res_da.append(DialogueActItem('inform', 'platform',
+                                                          platform_res.platform))
+                                res_da.append(DialogueActItem('inform', 'track',
+                                                          platform_res.track))
+                                if platform_info.train_name != 'none':
+                                    res_da.append(DialogueActItem('inform',
+                                                                  'train_name',
+                                                                  platform_info.train_name))
+                                else:
+                                    res_da.append(DialogueActItem('inform', 'direction',
+                                                          platform_res.direction))
                             else:
+                                res_da.append(DialogueActItem('inform', 'platform',
+                                                          'none'))
+                                res_da.append(DialogueActItem('inform', 'track',
+                                                          'none'))
                                 res_da.append(DialogueActItem('inform', 'direction',
-                                                      platform_res.direction))
-                        else:
-                            res_da.append(DialogueActItem('inform', 'platform',
-                                                      'none'))
-                            res_da.append(DialogueActItem('inform', 'track',
-                                                      'none'))
-                            res_da.append(DialogueActItem('inform', 'direction',
-                                                      platform_res.direction))
+                                                          platform_res.direction))
+                    except MethodNotSupported:
+                        res_da.append(DialogueActItem('inform', 'get_platform_not_supported'))
```

Comment:
I'd make it just general "not_supported" (with a general template "Tato funkce není dostupná").